src/main/resources/avro/referencemethods.avdl

```diff
@@ -12,19 +12,16 @@ as JSON.
 */
 record SearchReferenceSetsRequest {
   /**
-  If nonempty, return the reference sets which match any of the given
-  `md5checksum`s. See `ReferenceSet::md5checksum` for details.
+  If not null, return the reference sets which match the given
+  `md5checksum`. See `ReferenceSet::md5checksum` for details.
   */
-  array<string> md5checksums = [];
+  union { null, string } md5checksum = null;
 
   /**
-  If nonempty, return reference sets for which the accession
-  matches this string. Best to give a version number (e.g. `GCF_000001405.26`).
-  If only the main accession number is given then all records with
-  that main accession will be returned, whichever version.
-  Note that different versions will have different sequences.
+  If not null, return reference sets for which the accession
+  matches this string.
```

Comment:
This text implies that it is aware of accession version
numbering and knows how to parse it.  This is very reasonable.
Matching simple sub-strings, which was removed from the API, is
dangerous, as it could return spurious matches.

David Glazer <notifications@github.com> writes:
> In src/main/resources/avro/referencemethods.avdl:
> 
> >
> >    /**
> > -  If nonempty, return reference sets for which the accession
> > -  matches this string. Best to give a version number (e.g. `GCF_000001405.26`).
> > -  If only the main accession number is given then all records with
> > -  that main accession will be returned, whichever version.
> > -  Note that different versions will have different sequences.
> > +  If not null, return reference sets for which the accession
> > +  matches this string.
> 
> Does "matches this string" mean "is equal to this string" or "contains this
> substring" ? (I think I know the intent, but given that we're using the same
> phrase before and after this PR, I'm not sure.)
> 
> —
> Reply to this email directly or view it on GitHub.*
>